src/organisms/TopBar/Account/Account.styles.ts

```diff
@@ -34,6 +34,8 @@ export const RolesContainer = styled.section`
   flex-wrap: wrap;
   justify-content: center;
   max-width: 350px;
+  max-height: 200px;
+  overflow-y: auto;
```

Comment:
```suggestion
  overflow-y: auto;
  overflow-x: hidden;
```

Should probably have overflow x hidden here to prevent horizontal scrollbar from showing